docs/source/dev/actions_and_webhooks.rst

```diff
@@ -16,14 +16,18 @@ They are:
 
 #.  ``publish-to-pypi.yaml``, runs when a new version tag is pushed and publishes that tag version to PyPI.
 
+#. ``figshare.yaml``, releases new versions to `Figshare <https://orda.shef.ac.uk/articles/software/CITCOM_Software_Release/24427516?file=43686015>`_.
+
+#. ``publish-to-dafni.yaml``, which containerises and uploads every major release to `DAFNI <https://www.dafni.ac.uk/>`_.
+
+
 Webhooks
 ---------------
 
-The project also uses 3 `Webhooks <https://docs.github.com/en/webhooks-and-events/webhooks/about-webhooks>`_, which can
+The project also uses 2 `Webhooks <https://docs.github.com/en/webhooks-and-events/webhooks/about-webhooks>`_, which can
```

Comment:
There are 3 webhooks in the directory still, whats the reason for dropping to 2 in the docs?

Reply:
@christopher-wild Codacy has been deprecated since 2021 in our settings, so we don't actually use it at all (I asked about this in our chat a couple of weeks ago when you were on leave I think)